Lib/test/test_posix.py

```diff
@@ -22,6 +19,15 @@
 import textwrap
 from contextlib import contextmanager
 
+# Skip these tests if there is no posix or nt module.
+try:
+    import posix
+except ImportError:
+    try:
+        import nt as posix
```

Comment:
That's strange. For me, "test_posix" are tests for the POSIX platforms: all platforms but Windows.

If you want to write tests on alls platform including Windows, add tests to test_os, no?

Reply:
They are already written, and it turned out that they are compatible with Windows.

In the past there were several platform specific implementations: `posix`, `nt`, `dos`, `os2`, `mac`, `ce`. But `posix` and `nt` became so close that they are now build from the same source, and other implementations are gone.